src/main/java/jdbchomework/console/SkillsMenu.java

```diff
@@ -0,0 +1,64 @@
+package jdbchomework.console;
+
+import jdbchomework.controller.SkillController;
+import org.slf4j.LoggerFactory;
+
+public class SkillsMenu extends AbstractMenu {
+
+    private static final String SKILL_NOT_FOUND = "Skill not found";
+    private static final String PLEASE_ENTER_SKILL_NAME = "Please enter skill name: ";
+    private static final String PLEASE_ENTER_SKILL_ID = "Please enter skill ID: ";
+
+    private SkillController skillController;
+
+    public SkillsMenu(MenuContents menuContents, SkillController skillController) {
+        super(LoggerFactory.getLogger(SkillsMenu.class), menuContents);
+        this.menuContents = menuContents;
+        this.skillController = skillController;
+    }
+
+    public void menu() {
+        while (true) {
+            printListInConsole(menuContents.getServiceHeaderSkill(),
+                menuContents.getServiceSkillsMenu());
+            int action = readInt(CHOOSE_ACTION);
+            switch (action) {
+                case 1:
+                    printListInConsole(null, skillController.getAllSkills());
+                    break;
+                case 2:
+                    String name = readString(PLEASE_ENTER_SKILL_NAME);
+                    skillController.addSkill(name);
+                    System.out.println(name + " successfully added");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)